src/depool.rs

```diff
@@ -699,3 +766,162 @@ async fn encode_transfer_stake(dest: &str, amount: u64) -> Result<String, String
         "amount": amount
     })).await
 }
+
+async fn call_contract(
+    conf: Config,
+    wallet: &str,
+    depool: &str,
+    value: &str,
+	keys: &str,
+    body: &str,
+    answer_is_expected: bool
+) -> Result<(), String> {
+    if conf.no_wait_for_answer {
+        send_with_body(conf.clone(), wallet, depool, value, keys, body).await
+    } else {
+        call_contract_and_get_answer(conf.clone(), wallet, depool, value,
+            keys, body, answer_is_expected).await
```

Comment:
In Rust you should format calls like:
```
foo(
  arg1,
  arg2,
  ... ,
);
```
or in one line.

Reply:
Fixed